### PR TITLE
community : Update url_playwright from document_loaders to maximize efficiency retrieval

### PR DESCRIPTION
**Pull request by Kev744**
**Description :**
I added some modification to keep only some selectors in order to maximize execution time when loading documents and an another feature is to bypass human verification from some websites which has this ability (e.g. Cloudflare)

### CODE REVIEW

File: libs/community/langchain_community/document_loaders/url_playwright.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ async def evaluate_async(
 class UnstructuredHtmlEvaluator(PlaywrightEvaluator):
     """Evaluate the page HTML content using the `unstructured` library."""
 
-    def __init__(self, remove_selectors: Optional[List[str]] = None):
+    def __init__(self, remove_selectors: Optional[List[str]] = None, keep_only_selectors: Optional[List[str]] = None):
```
Comment (jacoblee93) [inline libs/community/langchain_community/document_loaders/url_playwright.py:62]: This name is misleading - I was expecting it to be a boolean.

How about `scraped_selectors` or something similar?

File: libs/community/langchain_community/document_loaders/url_playwright.py [hunk 2]
```diff
@@ -80,6 +81,15 @@ def evaluate(self, page: "Page", browser: "Browser", response: "Response") -> st
             for element in elements:
                 if element.is_visible():
                     element.evaluate("element => element.remove()")
+        
+        selectors = ','.join(self.keep_only_selectors) or 'body'
```
Comment (jacoblee93) [inline libs/community/langchain_community/document_loaders/url_playwright.py:85]: Will this technically miss out on things like `<head>` or `<meta>`?

File: libs/community/langchain_community/document_loaders/url_playwright.py [hunk 3]
```diff
@@ -135,8 +153,11 @@ def __init__(
         continue_on_failure: bool = True,
         headless: bool = True,
         remove_selectors: Optional[List[str]] = None,
+        keep_only_selectors: Optional[List[str]] = None,
         evaluator: Optional[PlaywrightEvaluator] = None,
         proxy: Optional[Dict[str, str]] = None,
+        header: Optional[str] = ('Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) '
```
Comment (jacoblee93) [inline libs/community/langchain_community/document_loaders/url_playwright.py:159]: This is unrelated, correct?

Let's call this `user_agent`.

Comment (Kev744) [reply libs/community/langchain_community/document_loaders/url_playwright.py]: Yes it's un related. It's done.

File: libs/community/langchain_community/document_loaders/url_playwright.py (hunk 2 above)
Comment (Kev744) [reply libs/community/langchain_community/document_loaders/url_playwright.py]: `<head>` and `<meta>` stays in the document because I only change body and not the head, so unstructured can partition these two tags even if he will not extract some informations from it (I think)

**Comment by Kev744** [comment]
@baskaryan Now he must work :) Thanks

**Comment by Kev744** [comment]
Can you update it, now it will work. Thanks you in advance @baskaryan @jacoblee93
